notifications_utils/polygons.py

```diff
@@ -243,9 +278,7 @@ def estimated_area(self):
         approximate conversion of degrees to square miles for UK
         latitudes, rather than a projection.
         '''
-        return sum(
-            polygon.area for polygon in self
-        ) * self.square_degrees_to_square_miles
+        return sum(polygon.area for polygon in self)
```

Comment:
so here we were operating in miles, but now we operate in metres, is it something that could cause trouble?

Reply:
I think the only place which uses the value as an absolute number is the admin app here:
https://github.com/alphagov/notifications-admin/blob/9ba37249a47f08b1396e63a34bc591c781739686/app/templates/views/broadcast/macros/area-map.html#L6

We’ll need to change it to do its own conversion from square metres to square miles

Reply:
cool stuff, do we have this written on a ticket or somewhere so we don't forget?

Reply:
I’ve got a work in progress PR on admin which will address the breaking changes: https://github.com/alphagov/notifications-admin/pull/3994

Reply:
Awesome stuff 💖